frb_example/pure_dart/rust/src/data.rs

```diff
@@ -16,6 +18,34 @@ struct PrivateData {
     lifetime: &'static str,
 }
 
+#[derive(Debug)]
+pub struct HideSyncData {
```

Comment:
May I know what is the difference between HideData and HideSyncData?

Reply:
analogue of `HideData` but not Send

corrected
```
#[derive(Debug)]
pub struct HideSyncData {
    content: String,
    box_content: Option<Rc<PrivateData>>,
}
```

Reply:
Then shall we rename it to something like `NonSendHideData` etc to make readers clear